x-pack/plugins/siem/public/pages/case/components/all_cases/columns.tsx

```diff
@@ -150,14 +150,26 @@ export const getCasesColumns = (
         },
       },
   {
-    name: i18n.SERVICENOW_INCIDENT,
+    name: i18n.EXTERNAL_INCIDENT,
     render: (theCase: Case) => {
       if (theCase.id != null) {
         return <ServiceNowColumn theCase={theCase} />;
```

Comment:
nit: Would you mind if we change the naming of the component ServiceNowColumn to ExternalServiceColumn